.github/workflows/linux.yml

```diff
@@ -73,8 +73,8 @@ jobs:
           export LIBOQS_SRC_DIR=`pwd`/liboqs && \
           ! pip3 install -r oqs-template/requirements.txt 2>&1 | grep ERROR && \
           python3 oqs-template/generate.py
-      - name: Full re-build
-        run: rm -rf _build && ./scripts/fullbuild.sh
+      - name: Full re-build & test
+        run: rm -rf _build && OPENSSL_BRANCH=${{ matrix.ossl-branch }} LIBOQS_BRANCH=main OQS_LIBJADE_BUILD=${{ matrix.libjade-build }} ./scripts/fullbuild.sh && ./scripts/runtests.sh
```

Comment:
I'm not opposed to this addition—just want to make sure it didn't "sneak in" to the PR (since I'm not immediately seeing the connection to dTLS).

Reply:
Thanks for the diligence @SWilson4 . No connection. Just work reduction (fewer PRs for me and anyone willing to review one):

Witnessing at how little time oqs-provider-committers can spend even looking at the code this seems vindicated (point in case: 3 days (in my time zone) to get a review unblocking merge... and in this case, getting ready for release -- even after mentioning this (pretty small) PR as waiting for review in a meeting: Pretty unsatisfactory and frustrating for me personally.

I understand this PR does not directly benefit the LF interests but primarily serves the wider FOSS community, so I maybe dare not complain. But serving the latter was always my priority with this project, so please understand my unhappiness.

To mitigate this unhappiness (and to also remove myself as a progress-"limiting factor" when busy with other things) what about dropping the one-review-to-merge rule for this project to allow it to move again at the pace it had before LF took control?

A second benefit of this would be to more widely document the true nature of this project: It should be a prototyping and test bed for as many users and projects as possible and not be considered "product code": OpenSSL has (OK, begins to have) much better code for the standard algorithms so I don't see the need to keep chasing what I by now deem a mirage given the still small (as exemplified in this case) support base. Objections to this change, @open-quantum-safe/oqs-provider-committers ?